src/test/java/org/apache/cassandra/sidecar/utils/SSTableImporterTest.java

```diff
@@ -67,7 +67,7 @@ public void setup() throws InterruptedException
         mockTableOperations1 = mock(TableOperations.class);
         TableOperations mockTableOperations2 = mock(TableOperations.class);
         when(mockConfiguration.getSSTableImportPollIntervalMillis()).thenReturn(10);
-        when(mockMetadataFetcher.delegate("127.0.0.1")).thenReturn(mockCassandraAdapterDelegate1);
```

Comment:
can we undo the changes to this file? They seem unnecessary and unrelated to the integration tests. Any reason for these changes?